community-tutorials/web/01-en.md

```diff
@@ -0,0 +1,427 @@
+Now you can open YOURLS over https://DOMAIN/admin and follow the shown steps and login
+
+### Step 2.2.1 - Installing a mordern YOURLS Theme (Sleeky) - Optional
+Download the Source code (zip) from https://github.com/Flynntes/Sleeky/releases/latest and unpack it and upload the `sleeky-backend` folder of the folder from the archiv in the user/plugin folder of your YOURLS installation <br>
+Now go to https://DOMAIN/admin and activate the sleeky plugin
+
+### Step 2.2.2 - Update YOURLS
+Switch to the folder and run:
+```sh
+git fetch origin
+git reset --hard origin/master
+```
+
+## Step 2.3 - Installing Nextcloud
+Nextcloud is an open source cloud software <br>
+Follow step 1.5 and create a new domain or subdomain for the website <br>
+Now go in to the folder you just created for it (Replace FOLDER with the name of the folder)
+```sh
+cd /var/www/FOLDER
+curl -L -o setup-nextcloud.php https://download.nextcloud.com/server/installer/setup-nextcloud.php
+```
+Now create a database see step 1.7 <br>
+Now you can open Nextcloud over https://DOMAIN/setup-nextcloud.php and follow the shown steps <br>
+When you are asked for an database select MySQL/MariaDB and enter the database login data <br>
+After you installed Nextcloud you should adjust some more settings
+```sh
+nano .htaccess
+```
+Now go to the end of the file and add: `Header always set Strict-Transport-Security "max-age=15552000"` <br>
+Now save your edit with <kbd>ctrl + x</kbd>, then press <kbd>y</kbd> and now save it with enter <br/>
+Now open the next file...
+```sh
+nano config/config.php
+```
+... and add after `'installed' => true,`
+```
+
+  'memcache.local' => '\OC\Memcache\APCu',
+  'default_phone_region' => 'DE',
+```
+Change DE if you do not live in Germany, look [here](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements) for the ISO code of your country <br/>
+Now save your file with <kbd>ctrl + x</kbd>, then press <kbd>y</kbd> and now save it with enter <br/>
+Now you haved finished the Nextcloud installation
+
+## Step 2.4 - Installing Uptime Kuma
+Uptime Kuma is an open source monitoring tool - [here is a demo of it](https://demo.uptime.kuma.pet/)<br>
+This tutorial is based on https://github.com/louislam/uptime-kuma/blob/master/README.md <br>
+Install Node.js 16
+```sh
+curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
+apt-get install nodejs -y
+npm config set fund false --global
+npm i npm -g
+{ crontab -l 2>/dev/null; echo "$(( $RANDOM % 60 )) $(( $RANDOM % 3 + 3 )) * * * sudo npm i npm -g" ; } | crontab -
+```
+Now install Uptime Kuma
+```sh
+cd /var/www/
+git clone https://github.com/louislam/uptime-kuma.git
+cd uptime-kuma
+npm run setup
+screen -AmdS uk node server/server.js
+{ crontab -l 2>/dev/null; echo "@reboot sleep 10 && cd /var/www/uptime-kuma && screen -AmdS uk node server/server.js" ; } | crontab -
+```
+Now you can open Uptime Kuma over http://IP:3001/ or http://DOMAIN:3001/
```

Comment:
Please add an info that it is needed to open port 3001.

Reply:
done